pages/about.js

```diff
@@ -17,7 +18,57 @@ import {
 import { ContentSection } from "@components/content-section";
 import { PageHeader } from "@components/page-header";
 import { Type } from "@components/typography";
-const contributorsData = require("nteract-members");
+
+type Member = {
+  name: ?string,
+  login: string,
+  websiteUrl: ?string,
+  avatarUrl: string,
+  url: string
+};
+
+async function getMembers(organisation: string) {
+  const token = process.env.GH_TOKEN;
+
+  if (!token) {
+    console.error("'GH_TOKEN' not set. Could not fetch nteract members.");
+    return [];
+  }
+
+  const client = new GraphQLClient("https://api.github.com/graphql", {
+    headers: {
+      Authorization: `Bearer ${token}`
+    }
+  });
+
+  const query = `{
+    organization(login: ${organisation}) {
+      members(first: 100) {
+        totalCount
+          nodes {
+            name
+            login
+            websiteUrl
+            avatarUrl
+            url
+          }
+      }
+    }
+  }`;
+
+  try {
+    const data = await client.request(query);
+    if (data.organization.members.totalCount > 100) {
+      console.error(
+        "100+ members in the organization. That's too much for one GraphQL call."
```

Comment:
Oh ha, we have more than 100 members so I'm not too surprised by this. 😅

Reply:
The token only has public scope. Since not all of our members list the membership as public the query returns less than 100 members